README.md

```diff
@@ -1,15 +1,35 @@
-# data-structures
+#data-structures
 
-This repository holds sample code for a number of data structures implemented with python.
+###This repository holds sample code for a number of data structures implemented with python.
+
+*Feb 4th, 2015* - Added a Linked List structure in Python. Has classes for each node, as well as the structure itself. Includes functionality for:
+                      <ul>
+                      <li>insert(value): Inserts the given value at the front of the list</li>
+                      <li>pop(): Removes the front value from the list and returns that to the user</li>
+                      <li>size(): Returns the size (number of nodes) in a list</li>
+                      <li>search(value): Searches a list for a given value, if found, returns the node</li>
+                      <li>remove(node): If the given node exists, removes it from the list.</li>
+                      <li>display(): Displays the string formatted list as a tuple.</li>
+                      <li>__repr__: Same as display, but can be called by just inputting the name of the list</li>
+                      </ul>
+                  <p>Each of these functions has tests to ensure they are working as expected.</p>
+*Feb 5th, 2015* - Added a stack structure in Python. Has a class for an element of the stack, as well as the stack stucture itself. Includes functionality for:
+                      <ul>
+                      <li>push(value): Adds the value at to top of the stack</li>
+                      <li>pop(): Removes the top element from the stack and returns it.</li>
+                      </ul>
+                  <p>Each of these functions has tests to ensure they are working as expected.</p>
 
-20150204 - Linked list add
 
 Collaborations:
 Constantine Hatzis and Nick Becker
 
-Resources:
-Cris Ewing
-Mark Ableidinger
-stackoverflow.com
-tutorialspoint.com/python
 
+Resources:
+<ul>
+<li>Cris Ewing</li>
+<li>Mark Ableidinger</li>
+<li>stackoverflow.com</li>
+<li>tutorialspoint.com/python</li>
+<li>docs.python.org</li>
+</ul>
```

Comment:
gah,  don't write html in markdown.  just write proper markdown.  Work at it until you get it right.